refactor: remove window.disableShortcuts

PierreDemailly opened the pull request.

targos commented:
I don't think it's being handled in the right place. It should be the shortcut handler that checks whether the event target is an input element or similar, and do nothing when it's the case.

PierreDemailly commented:
> I don't think it's being handled in the right place. It should be the shortcut handler that checks whether the event target is an input element or similar, and do nothing when it's the case.

I agree, mb 👍 I removed `window.disableShortcuts` (and changed PR title), it works better now
